docs/plutus-ir/plutus-ir.md

```diff
@@ -0,0 +1,193 @@
+# Plutus IR design proposal
+
+## Syntax
+
+The syntax is mostly as in Plutus Core. Where we modify the syntax of Plutus Core we will
+list only the additions and removals, marking them explicitly with `+` or `-`.
+We also require syntax with lists of subparts. We denote a list of `t`-productions with `t..`.
+
+```
+VarDecl VD      := (vardecl x T)
+TyVarDecl TVD   := (tyvardecl alpha K)
```

Comment:
Well, they bind variables.

Reply:
The Plutus Core spec calls them variables.

Reply:
It is a variable in the sense of a variable in math (not C).

Reply:
Right, but those are bound variables in a formula. I suppose these are too, but it's less connected.

Reply:
In the context of FP, these are always called variables iinm.

I'm going to resolve this convo since I think the terminology is somewhat orthogonal to the IR.